### PR TITLE
Sftp server

### PR DESCRIPTION
**Pull request by ahalberkamp**
In this pr a new chart was created to easily deploy a sftp server to k8s. 
This charts is using this image: https://github.com/atmoz/sftp

### CODE REVIEW

File: charts/sftp/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    -f /path/to/shop/values/my-shop-staging/sftp-secret.yaml 
+    . 
+```
+
+## Assigning a external IP to the SFTP
+
+To assign a static ip to the sftp please execute
+
+```bash
+ kubectl expose deployment <service- name> --type=LoadBalancer --name=<loadbalancer nmae -n <namespace> --load-balancer-ip='<static ip>'
```
Comment (ahmetoz) [inline charts/sftp/README.md]:
```suggestion
 kubectl expose deployment <service-name> --type=LoadBalancer --name=<loadbalancer-name> -n <namespace> --load-balancer-ip='<static ip>'
```

File: charts/sftp/README.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+```bash
+$ helm upgrade --install \
+    --namespace my-shop-namespace \
+    my-sftp-custom-release-name \
+    -f /path/to/shop/values/my-shop-staging/values.yaml \
+    -f /path/to/shop/values/my-shop-staging/sftp-secret.yaml 
+    . 
+```
+
+## Assigning a external IP to the SFTP
```
Comment (ahmetoz) [inline charts/sftp/README.md]:
```suggestion
## Assigning an external IP to the SFTP
```

File: charts/sftp/README.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+To assign a static ip to the sftp please execute
```
Comment (ahmetoz) [inline charts/sftp/README.md]:
```suggestion
To assign a static ip to the SFTP, please execute the following command:
```

File: charts/sftp/README.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+$ helm upgrade --install --namespace my-shop-namespace my-service -f values.yaml .
+```
+
+### Example
+
+Given that you are in the current kubernetes configuration directory 
+(where [Chart.yaml](Chart.yaml) and [templates/](templates/) are located).
+Commands are relative to this directory. 
+
+Install service using custom values and secrets:
```
Comment (ahmetoz) [inline charts/sftp/README.md]:
```suggestion
Execute following helm command to create SFTP service:
```

File: charts/sftp/README.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+```bash
+$ helm upgrade --install --namespace my-shop-namespace my-service -f values.yaml .
+```
+
+### Example
+
+Given that you are in the current kubernetes configuration directory 
+(where [Chart.yaml](Chart.yaml) and [templates/](templates/) are located).
+Commands are relative to this directory.
```
Comment (ahmetoz) [inline charts/sftp/README.md]:
```suggestion
Note that, following commands are relative to the working directory:
```

File: charts/sftp/README.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+| `sshConfig.ssh_host_ed25519_key_pub`| public ed25519-key                        | ``                                          | 
+| `sshConfig.ssh_host_rsa_key`       | private rsa-key                            | ``                                          | 
+| `sshConfig.ssh_host_rsa_key_pub`   | public rsa-key                             | ``                                          | 
+| `service.port`                     | Port to expose Service                     | `22`                                        |
+| `service.nodePort`                 | nodePort of the  Service                   | `31000`                                     |
+| `persistentVolume.enabled`         | If true, use persistent volume             | `true`                                      |
+| `persistentVolume.annotations`     | annotations put on the volume              | `{}`                                        |
+| `persistentVolume.accessModes`     | access modes for volume                    | `[ReadWriteOnce]`                           |
+| `persistentVolume.existingClaim`   | If set, use existing PVC                   | `""`                                        |
```
Comment (ahmetoz) [inline charts/sftp/README.md]: Use existing PVC: what does it mean ?

File: charts/sftp/README.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+  - [Assigning a external IP to the SFTP](#assigning-a-external-ip-to-the-sftp)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+## Prerequisites
+
+* Kubernetes 1.8+ with beta APIs enabled
+
+## Chart Details
+
+This chart will do the following:
```
Comment (ahmetoz) [inline charts/sftp/README.md:24]: Can we describe here that chart uses the atmoz/sftp:latest image to do the job?

File: charts/sftp/README.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
+# Sftp Helm Chart
+
+Helm Chart for setting up a sftp server
```
Comment (ahmetoz) [inline charts/sftp/README.md]:
```suggestion
Helm Chart for setting up an sftp server
```

File: charts/sftp/README.md [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+
+To install the chart with the release name `my-service` using a dedicated namespace(recommended):
+
+```
+$ helm upgrade --install --namespace my-shop-namespace my-service -f values.yaml .
+```
+
+The chart can be customized using the following configurable parameters:
+
+|          Parameter                 |                Description                 |                   Default                   |
```
Comment (ahmetoz) [inline charts/sftp/README.md]: Is there any required parameters, if yes I think we should mark those as required ?

**Review by ahmetoz — CHANGES_REQUESTED** [review]
Good work 👍🏽 could you please check the linting issue on build ?

File: charts/sftp/README.md (hunk 7 above)
Comment (ahalberkamp) [reply charts/sftp/README.md:24]: 3fabf38

File: charts/sftp/README.md (hunk 9 above)
Comment (ahalberkamp) [reply charts/sftp/README.md]: 87ec092

File: charts/sftp/README.md (hunk 6 above)
Comment (ahalberkamp) [reply charts/sftp/README.md]: de81924

**Review by ahmetoz — APPROVED** [review]

**Pull request merged**